### ISSUE TITLE
Outdated instructions

### ISSUE THREAD

**Issue by KrisSoto** [magento/devdocs#1228]
Feedback on page: /guides/v2.1/install-gde/prereq/apache.html

This is outdated, this is based on neither the newest version of ubuntu or apache. Following these instructions will NOT work anymore.

**Comment by jeff-matthews** [comment]
Thanks for the feedback @KrisSoto. Are you referring to a particular section? For example, installing Ubuntu, enabling rewrites, upgrading, troubleshooting?

**Comment by KrisSoto** [comment]
The issue I had was with setting up Apache, when I went to research the issues I was having I found that the config file mentioned is no longer current. When I tried to make the changes to the proper file, it didn't have all of the same information and didn't allow me to continue setup. I can give a more detailed response later on if necessary when I'm back at my laptop.

On Jun 19, 2017 4:36 PM, Jeff Matthews <notifications@github.com> wrote:

Thanks for the feedback @KrisSoto<https://github.com/krissoto>. Are you referring to a particular section? For example, installing Ubuntu, enabling rewrites, upgrading, troubleshooting?
